avocado/utils/process.py

```diff
@@ -1189,4 +1194,6 @@ def system_output(cmd, timeout=None, verbose=True, ignore_status=False,
     cmd_result = run(cmd=cmd, timeout=timeout, verbose=verbose, ignore_status=ignore_status,
                      allow_output_check=allow_output_check, shell=shell, env=env,
                      sudo=sudo, ignore_bg_processes=ignore_bg_processes)
+    if strip_trail_nl:
```

Comment:
@ggopinat my mistake, I did not notice that `strip_trail_nl` is set to `True` by default.